app/scripts/directives/logViewer.js

```diff
@@ -45,7 +47,7 @@ angular.module('openshiftConsole')
         scope: {
           followAffixTop: '=?',
           followAffixBottom: '=?',
-          resource: '@',
+          resource: '=',
```

Comment:
Maybe call this `object` to avoid confusion with resource strings or resourceGroupVersion objects?